bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties

```diff
@@ -35,29 +35,32 @@ thing-type.config.tr064.fritzbox.rejectedCallDays.label = Rejected Call Days
 thing-type.config.tr064.fritzbox.rejectedCallDays.description = List of days for which rejected calls should be calculated.
 thing-type.config.tr064.fritzbox.tamIndices.label = TAM
 thing-type.config.tr064.fritzbox.tamIndices.description = List of answering machines (starting with 0).
-thing-type.config.tr064.fritzbox.timeout.label = Timeout
-thing-type.config.tr064.fritzbox.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
 thing-type.config.tr064.fritzbox.user.label = Username
 thing-type.config.tr064.fritzbox.wanBlockIPs.label = WAN Block IPs
 thing-type.config.tr064.fritzbox.wanBlockIPs.description = List of IPs that can be blocked for WAN access.
 thing-type.config.tr064.generic.host.label = Host
 thing-type.config.tr064.generic.host.description = Host name or IP address.
 thing-type.config.tr064.generic.password.label = Password
 thing-type.config.tr064.generic.refresh.label = Refresh Interval
-thing-type.config.tr064.generic.timeout.label = Timeout
-thing-type.config.tr064.generic.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
 thing-type.config.tr064.generic.user.label = Username
 thing-type.config.tr064.subdevice.refresh.label = Refresh Interval
 thing-type.config.tr064.subdevice.uuid.label = UUID
 thing-type.config.tr064.subdevice.uuid.description = UUID of the sub-device
 thing-type.config.tr064.subdeviceLan.macOnline.label = MAC Online
-thing-type.config.tr064.subdeviceLan.macOnline.description = List of MACs for "online" status detection (format: 11:11:11:11:11:11).
+thing-type.config.tr064.subdeviceLan.macOnline.description = List of MACs for "online" status detection (format: 11:11:11:11:11:11). One MAC-Address per row.
 thing-type.config.tr064.subdeviceLan.refresh.label = Refresh Interval
 thing-type.config.tr064.subdeviceLan.uuid.label = UUID
 thing-type.config.tr064.subdeviceLan.uuid.description = UUID of the sub-device
 
 # thing types config
 
+thing-type.config.tr064.fritzbox.timeout.label = Timeout
+thing-type.config.tr064.fritzbox.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
+thing-type.config.tr064.generic.timeout.label = Timeout
+thing-type.config.tr064.generic.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
```

Comment:
How did they end here? Perhaps no longer referenced?

Reply:
If they are no longer referenced, i would suspect that the i18n tool would remove them.
I only ran the i18n mvn command and this is the result. As far as i can see there is no real change, the lines are only moved to a different place in the file. So i would suggest to merge it as is. Please advice if you have other suggestions.

Reply:
You can try to delete this section and re-run the tool. If they don't reappear, they are no longer referenced.

Reply:
Done. Seems there where more issues, i completely remove all contents and re-run the tool. It cleaned more entries.